backend/app.py

```diff
@@ -400,17 +401,20 @@ def db_refresh_route():
 
     Example working cURL:
     (well, it works for @joeflack4 when actually putting in the token itself. With ${GH_LIMITED_PERSONAL_ACCESS_TOKEN},
-    it should work, but gives "curl: (16) Error in the HTTP2 framing layer"
-
-    curl -X POST \
-      -H "Authorization: Bearer ${GH_LIMITED_PERSONAL_ACCESS_TOKEN}" \
-      -H "Accept: application/vnd.github.v3+json" \
-      -H "Content-Type: application/json" \
-      -d '{"event_type": "refresh-db"}' \
-      https://api.github.com/repos/jhu-bids/TermHub/dispatches
-    """
-    print("to be implemented")
+    it should work, but gives "curl: (16) Error in the HTTP2 framing layer" """
 
+    headers = {
+        "Authorization": f"Bearer {CONFIG['personal_access_token']}",
+        "Accept": "application/vnd.github.v3+json",
+        "Content-Type": "application/json"
+    }
+    args = {"event-type":"refresh-db"}
+    url = 'https://api.github.com/repos/jhu-bids/TermHub/dispatches'
+    payload = {
+        'event_type': 'refresh-db',
+    }
+    response = requests.post(url, headers = headers,data=json.dumps(payload))
```

Comment:
From what I see, it looks like this worked correctly! https://github.com/jhu-bids/TermHub/actions